src/Controls/src/Core/Platform/Windows/Extensions/PointExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Maui.Graphics;
+using WPoint = Windows.Foundation.Point;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	public static class PointExtensions
+	{
+		public static WPoint ToWindows(this Point point)
```

Comment:
I think this is ToNative in Core or ToPoint in Graphics

Reply:
Marked it obsolete